apps/playground/src/components/playground/chat-page-client.tsx

```
@@ -136,6 +147,21 @@ export default function ChatPageClient({
 			onFinish: async ({ message }) => {
 				isNewChatRef.current = false;
 
+				// Debug: Log the full message structure
+				console.log("=== onFinish message ===");
```

ESLint

[error] Unexpected console statement.

(no-console)